tests/openwisp2/settings.py

```diff
@@ -148,7 +148,6 @@
         },
         'py.warnings': {'handlers': ['console'], 'propagate': False},
         'celery': {'handlers': ['console'], 'level': 'DEBUG'},
-        'celery.task': {'handlers': ['console'], 'level': 'DEBUG'},
```

Comment:
One last quesiton: why do we need to remove this? Don't remember if we already discussed, sorry.

Reply:
@nemesisdesign Actually I considered removing the submodule logger because the version 5 docs say that they have deprecated the celery submodules like celery.task ( [reference](https://docs.celeryproject.org/en/stable/whatsnew-5.0.html#removed-deprecated-modules) ) . The tests work fine either way round, keeping or removing the logger didn't pose any errors. I was a bit confused at first myself whether it is really necessary to remove the logger, and hence wanted some reviews on it. Please let me know your thoughts about it, I'll make any changes accordingly. Thanks!